netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -104,12 +104,19 @@
   private static final Status EXHAUSTED_STREAMS_STATUS =
           Status.UNAVAILABLE.withDescription("Stream IDs have been exhausted");
 
+  private static final int BDP_MEASUREMENT_PING = 1234;
+  private static final int MAX_WINDOW_SIZE = 8 * 1024 * 1024;
+
   private final Http2Connection.PropertyKey streamKey;
   private final ClientTransportLifecycleManager lifecycleManager;
   private final Ticker ticker;
   private final Random random = new Random();
   private WriteQueue clientWriteQueue;
   private Http2Ping ping;
+  private boolean pinging;
+  private int pingcount;
+  private int pingreturn;
+  private int dataSizeSincePing;
```

Comment:
@mfcripps how about putting all of these fields into an inner class and move them to `AbstractNettyHandler`? Also we use camel case, `pingCount`, `pingReturn` and so :-).